plone/outputfilters/filters/resolveuid_and_caption.py

```diff
@@ -324,6 +324,7 @@ def unknown_starttag(self, tag, attrs):
                 scheme = urlsplit(href)[0]
                 if not scheme and not href.startswith('/') \
                         and not href.startswith('mailto<') \
```

Comment:
Totally right but I thought it was some kind of super special case because it actually works somehow.